CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Added
 
-- Nothing.
+- Default behavior changed when malformed is pushed into toUtf8.  Rather than dumping the bad string in an exception (dangerous behavior), it will return a blank string.
```

Comment:
This is not dangerous behavior: it is documented and expected behavior.

Passing around an empty string when an invalid value to escape is passed in can cause much more serious issues.